### PR TITLE
patch only svc/svc_cx symbols

### PR DESCRIPTION
**Pull request by cbouvet-ledger**

### CODE REVIEW

File: src/launcher.c [hunk 1]
```diff
@@ -147,6 +149,23 @@ struct app_s *get_current_app(void)
...
 }
 
+static bool is_svc_inlined(void)
+{
+  switch (sdk_version) {
+  case SDK_BLUE_1_5:
+  case SDK_BLUE_2_2_5:
+  case SDK_NANO_S_1_5:
+  case SDK_NANO_S_1_6:
+  case SDK_NANO_S_2_0:
```
Comment (xchapron-ledger) [inline src/launcher.c:159]: I'm think this one is not inline?

File: src/launcher.c [hunk 2]
```diff
@@ -171,22 +190,46 @@ int replace_current_code(struct app_s *app)
...
+    if (app->elf.svc_call_addr != 0) {
+      uint32_t start = app->elf.svc_call_addr - text_start;
+      uint32_t code_length = 2;
+
+      if (patch_svc(memory.code + start, code_length) != 0) {
+        /* this should never happen, because the svc were already patched
+         * without error during the first load */
+        _exit(1);
+      }
+    }
```
Comment (xchapron-ledger) [inline src/launcher.c:215]: This block appears a lot, can you make a function with it?

File: src/launcher.c (hunk 1 above)
Comment (cbouvet-ledger) [reply src/launcher.c:159]: :shrug:

File: src/launcher.c (hunk 2 above)
Comment (fbeutin-ledger) [reply src/launcher.c:214]: +1

File: src/launcher.c [hunk 3]
```diff
@@ -147,6 +149,22 @@ struct app_s *get_current_app(void)
...
+  case SDK_NANO_S_1_5:
+  case SDK_NANO_S_1_6:
+  case SDK_NANO_X_1_2:
+    return true;
+    break;
+  default:
+    return false;
+    break;
+  }
+}
```
[the remark this answers is not in the document]
Comment (cbouvet-ledger) [reply src/launcher.c:165]: non, clang me chie dessus

**Pull request merged**